build.sbt

```diff
@@ -12,14 +14,21 @@ lazy val `demo-routeguide` = project
   .settings(Seq(
     organization := "frees-io",
     organizationName := "47 Degrees",
-    scalaVersion := "2.12.3",
+    scalaVersion := "2.12.4",
     resolvers ++= Seq(
       Resolver.sonatypeRepo("releases"),
       Resolver.bintrayRepo("beyondthelines", "maven")
     ),
+    scalacOptions := Seq("-Ywarn-unused-import"),
```

Comment:
This is overriding all the existing `scalacOptions`, let's change it for on `+=`.

Reply:
Addressed. Thanks for you help